clusterDeployer.py

```diff
@@ -121,7 +121,7 @@ def teardown_masters(self) -> None:
         self._ai.ensure_infraenv_deleted(f"{cluster_name}-x86_64")
         self._ai.ensure_infraenv_deleted(f"{cluster_name}-arm64")
 
-        self._local_host.bridge.remove_dhcp_entries(self._cc.master_vms())
+        self._local_host.remove_dhcp_entries(self._cc.workers)
```

Comment:
Didn't test but shouldn't this be `self._cc.masters`?

Reply:
It definitely should, thanks